docs/ui-components.md

```diff
@@ -0,0 +1,136 @@
+# UI Components
```

Comment:
add convention document explaining the structure of it and rules.
1. the display name
2. file name dash between words all lower case
3. class name is upper camel case corresponding to file name
4. CSS conventions
What about CSS?  think only issue is with volume which was also called volume-control in CSS - can you please verify and according to that we can decide - I would prefer to make the change in one cut and so better to handle this as well

Reply:
explain the disabled prop and usage